README.md

```diff
@@ -67,15 +68,67 @@ argument. Some useful kernel aguments:
 
 * `init=PROGRAM` - Specifies the userspace program for PID 1. Browse `./user`
   for currently available programs.
-* `test=TEST` - Requests the kernel to run the specified test (from `./tests`
-  directory).
+* test-related arguments described in the [following section](#test-infrastructure).
+
+Test infrastructure
+---
+
+##### User tests
+Located in `/user/utest`.
+User-space test function signature looks like this: `int test_[name](void)` 
+and should be defined in `user/utest/utest.h`.
+In order to make the test runnable one has to add one of these lines to `test/utest.c` file:
+* `UTEST_ADD_SIMPLE([name])` - test fails on assertion or non-zero return value.
+* `UTEST_ADD_SIGNAL([name], [SIGNUMBER])` - test passes when terminated with `[SIGNUMBER]`.
```

Comment:
I think, that square brackets are usually used for non-obligatory arguments.

Reply:
In this context I used it as some kind of pattern matching. But here you have it. I replaced it with curly braces.